tools/clang/tools/dxc/dxc.cpp

```diff
@@ -747,6 +767,7 @@ void DxcContext::Recompile(IDxcBlob *pSource, IDxcLibrary *pLibrary, IDxcCompile
     ConcatArgs.size(), ConcatDefines.data(),
     ConcatDefines.size(), pIncludeHandler, &pResult));
   *ppCompileResult = pResult.Detach();
+#endif // _WIN32
```

Comment:
Might be good to add an assert and set ppCompileResult to null in case someone calls this on Linux.

Reply:
Good idea. Fixed.